utils/perl5lib/Batch/BatchMaker.pm

```diff
@@ -887,11 +889,11 @@ sub overrideNodeCount()
 }
 
 #==============================================================================
-# Simple factory class to get the right BatchMaker class for each machine.  
-# The only downside to this strategy is that we have to have a BatchMaker_${machine} 
-# class for every machine we port. 
+# Simple factory class to get the right BatchMaker class for each machine.
```

Comment:
I'm getting overwhelmed by the number of whitespace changes in this PR. I'm scrolling through them fast and potentially missing real changes. I like that we're removing trailing whitespaces, but maybe our next PR should just remove them globally so we stop seeing them in every PR.